src/styles/pages/login.module.scss

```diff
@@ -0,0 +1,31 @@
+.logoLogin{
+  margin: 0 auto;
+  width: 25rem;
+}
+.box1 {
```

Comment:
box1, box2, box3 seem like bad class names - what are they for? It'd be nice to name these more descriptively